adoc/admin-security-certificates.adoc

```diff
@@ -701,11 +701,10 @@ You should receive the following files in return:
 [[self_signed_certificate]]
 === Self-signed Server Certificate
 
-[WARNING]
+[NOTE]
 ====
-Do not use self-signed certificates in production environments.
-There is severe risk of Man-in-the-middle attacks.
-Use proper trusted 3rd-party signed certificates.
+Make sure that in case you use self-signed certificates that the Certificate Authority
+used for signing is configured in a secure way as trusted Certificate Authority on the clients.
```

Comment:
This sentence is a bit too long and incoherent. Suggested rewording: 
"In the case that you decide to use self-signed certificates, make sure that the Certificate Authority
used for signing is configured securely as a trusted Certificate Authority on the clients."

Reply:
Updated and rebase onto maintenance/CaaS4 branch